api/customers/customersRouter.js

```diff
@@ -117,6 +78,31 @@ router.put('/:id', async (req, res) => {
   }
 });
 
+router.put('/remove_favorite-groomer/:id', async (req, res) =>{
+  try{
+    let customer = await customersModel.findById(req.params.id);
+    if(req.body){
+      if(customer.favorite_groomers !== null){
+        const filteredFavorites = customer.favorite_groomers.filter((fav) => fav !== req.body.favorite_groomers);
+        customer.favorite_groomers = filteredFavorites;
+      }
+      const updatedCustomer = await customersModel.update(
+        req.params.id,
+        customer
+      );
```

Comment:
Looks good, could probably add in some logic if the given id doesn't exist in customer.favorite_groomers but this looks like it should get the job done!

Reply:
Good call out, I'll fix it in a future PR. For now if the user doesn't exist its caught in the catch and says that the user is a null. Not the best error handling, but at least its not breaking the system.